_posts/en/newsletters/2022-11-23-newsletter.md

```diff
@@ -0,0 +1,127 @@
+name: 2022-11-23-newsletter
+slug: 2022-11-23-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+*No significant news this week was found on the Bitcoin-Dev or
+Lightning-Dev mailing lists.*
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [Did the P2SH BIP-0016 make some Bitcoin unspendable?]({{bse}}115803)
+  User bca-0353f40e lists 6 outputs that existed with the P2SH script format,
+  `OP_HASH160 OP_DATA_20 [hash_value] OP_EQUAL`, before [BIP16][]'s activation.
+  One of those outputs had been spent under the old rules before activation and
+  any remaining UTXOs are still spendable under BIP16's backward compatibility rules.
+
+- [What software was used to make P2PK transactions?]({{bse}}115962)
+  Pieter Wuille notes that P2PK outputs were created using the original Bitcoin
+  software in coinbase transactions as well as when sending using [pay-to-IP
+  address][wiki p2ip].
+
+- [Why are both txid and wtxid sent to peers?]({{bse}}115907)
+  Pieter Wuille references [BIP339][] and explains that while using wtxid is
+  better for relay (due to malleability among other reasons), some peers do not
+  support the newer wtxid identifiers and txids are supported for older
+  pre-BIP339 peers for backward compatibility.
+
+- [How do I create a taproot multisig address?]({{bse}}115700)
+  Pieter Wuille points out that Bitcoin Core's existing [multisig][topic multisignature] RPCs (like
+  `createmultisig` and `addmultisigaddress`) will only support legacy wallets
+  and outlines that with Bitcoin Core 24.0, users will be able to use
+  [descriptors][topic descriptors] and RPCs (like `deriveaddresses` and
+  `importdescriptors`) along with the new `multi_a` descriptor to create
+  [taproot][topic taproot]-compatible multisig scripts.
+
+- [Is it possible to skip IBD on pruned node?]({{bse}}116030)
+  While not currently supported in Bitcoin Core, Pieter Wuille points to the
+  [assumeutxo][topic assumeutxo] project which would allow for a new node to
+  bootstrap by using trusted snapshots of the UTXO set.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 24.0 RC4][] is a release candidate for the
+  next version of the network's most widely used full node
+  implementation.  A [guide to testing][bcc testing] is available.
```

Comment:
We could mention here that v24.0 was tagged as final five days ago: https://github.com/bitcoin/bitcoin/releases/tag/v24.0